src/util/edit.cc

```diff
@@ -70,8 +70,9 @@ void EditEngine::modifyBytes(size_t pos, const data::BinData& bytes,
   if (next_it == address_mapping_.cend() || end_pos <= next_it.key()) {
     // This is the case when whole query range is located in only one node.
     if (it->fragment_ == nullptr) {
-      // TODO(catsuryuu): little refactoring
-      if (address_mapping_.find(end_pos) == address_mapping_.cend()) {
+      if (next_it == address_mapping_.cend() || end_pos != next_it.key()) {
```

Comment:
Should I use `<` instead of `!=` for readability?

Reply:
Yup, `<` seems to be slightly better, but I don't have any strong preference here.